### ISSUE TITLE
Bug: XAML Controls Gallery: Accessibility: [MAS 4.2.1]: The narrator does not read the tooltip info using Caps+0 for the image available in the Tooltip window of the Xaml controls gallery.

### ISSUE THREAD

**Issue by v-amnat** [microsoft/microsoft-ui-xaml#4280]
**Tags**
Accessibility, A11yCT, A11yMAS, A11yLowImpact, MAS4.2.1, Narrator, Sev4, XAML Controls Gallery (Debug), Win32, Wipro

**Steps to reproduce the bug**
1. Launch the XAML Controls Gallery (Debug) App
2. Go to the Status and Info-> Tooltips-> Image.
3. Navigate to the image under "An image with a tooltip using placement react" heading
4. Use Caps+0 to read the tooltip of the image and observe the narrator announcement.

**Actual behavior**
The narrator does not read the tooltip info using Caps+0 for the image available in the  Tooltip window of the Xaml controls gallery.
Narrator says as no advanced info available while user hit on the Caps+0 keys to read the tooltip info of this image(i.e. Non occluding ToolTip)

**Expected behavior**
Using Caps+0 , narrator should read the tooltips info of the image.

**User Impact**
If the tooltips info available for the image is not being read by the narrator then, additional info could not be conveyed to the narrator user 

**Attachment**
[Narrator-Tooltip-NoAnnouncement_edit.zip](https://github.com/microsoft/microsoft-ui-xaml/files/6020713/Narrator-Tooltip-NoAnnouncement_edit.zip)

**Version Info**
Microsoft OS: 21313.1000
App: XAML Controls Gallery (Debug)

**Comment by StephenLPeters** [comment]
@stmoy FYi
